web3.md

```diff
@@ -1257,32 +1255,50 @@ Transaction Receipts
     rule <k> #eth_call
           => #pushNetworkState
           ~> #setMode NOGAS
-          ~> mkTX !ID:Int
-          ~> #loadNonce #parseHexWord(#getString("from", J)) !ID
-          ~> loadTransaction !ID J
-          ~> signTX !ID #parseHexWord(#getString("from", J))
-          ~> #prepareTx !ID #parseHexWord(#getString("from", J))
+          ~> #loadTx J
           ~> #eth_call_finalize
          ...
          </k>
          <params> [ ({ _ } #as J), TAG, .JSONs ] </params>
-      requires isString( #getJSON("to"   , J) )
-       andBool isString( #getJSON("from" , J) )
+      requires isString( #getJSON("from" , J) )
 
     rule <k> #eth_call => #rpcResponseError(-32027, "Method 'eth_call' has invalid arguments") ...  </k>
          <params> [ ({ _ } #as J), TAG, .JSONs ] </params>
       requires notBool isString( #getJSON("from", J) )
 
     syntax KItem ::= "#eth_call_finalize"
  // -------------------------------------
-    rule <k> #eth_call_finalize
+    rule <statusCode> EVMC_SUCCESS </statusCode>
+         <k> _:Int ~> #eth_call_finalize
           => #setMode NORMAL
           ~> #popNetworkState
           ~> #clearGas
           ~> #rpcResponseSuccess(#unparseDataByteArray( OUTPUT ))
          ...
-        </k>
+         </k>
          <output> OUTPUT </output>
+
+    rule <statusCode> EVMC_REVERT </statusCode>
+         <k> TXID:Int ~> #eth_call_finalize
+          => #setMode NORMAL
+          ~> #popNetworkState
+          ~> #clearGas
+          ~> #sendResponse( "error" :
+               { "message": "VM Exception while processing transaction: revert",
+                 "code": -32000,
+                 "data": {
+                     "0x" +String #hashSignedTx( TXID ): {
+                     "error": "revert",
+                     "program_counter": PCOUNT +Int 1,
+                     "return": #unparseDataByteArray( RD )
+                   }
+                 }
+               } )
```

Comment:
We shouldn't have direct calls to `#sendResponse`. Use one of the wrapper helpers or make a new wrapper helper.